src/ImageSharp/IImage.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using SixLabors.Primitives;
+
+namespace SixLabors.ImageSharp
+{
+    /// <summary>
+    /// Encapsulates the properties and methods that describe an image.
+    /// </summary>
+    public interface IImage : IImageInfo, IDisposable
+    {
+        /// <summary>
+        /// Gets the size of the image.
+        /// </summary>
+        /// <returns>The <see cref="Size"/></returns>
+        Size Size();
```

Comment:
We can push this up to `IImageInfo`.
Actually there are Width + Height properties here, so maybe it should be an extension method.